src/test/java/visualizations/HistogramsTest.java

```diff
@@ -0,0 +1,73 @@
+package visualizations;
```

Comment:
same as for PR: https://github.com/jethronap/JavaDataAnalysis/pull/12

Reply:
ok!